File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/DomainAllInfo.js
goog.provide('API.Client.DomainAllInfo');

/**
 * Complete domain registration information returned from the registrar, including contacts, nameservers, and expiry dates.
 * @record
 */
API.Client.DomainAllInfo = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.oPSVersion;

/**
 * @type {!API.Client.DomainAllInfo_attributes}
 * @export
 */
API.Client.DomainAllInfo.prototype.attributes;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.object;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.protocol;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.responseText;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.responseCode;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.action;

/**
 * @type {!string}
 * @export
 */
API.Client.DomainAllInfo.prototype.isSuccess;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/DomainAllInfo.scala
package io.swagger.server.model


/**
 * Complete domain registration information returned from the registrar, including contacts, nameservers, and expiry dates.
 *
 * @param _OPS_version 
 * @param attributes 
 * @param `object` 
 * @param protocol 
 * @param response_text 
 * @param response_code 
 * @param action 
 * @param is_success 
 */
case class DomainAllInfo (
  _OPS_version: Option[String],
  attributes: Option[DomainAllInfo_attributes],
  `object`: Option[String],
  protocol: Option[String],
  response_text: Option[String],
  response_code: Option[String],
  action: Option[String],
  is_success: Option[String]
)

